spec/helpers/pages_helper_spec.rb

```diff
@@ -27,6 +27,27 @@
     end
   end
 
+  describe 'format_ak_ui_url' do
+    it 'returns the passed value if no Settings.ak_ui_url' do
+      Settings.ak_ui_url = nil
```

Comment:
I only see `AK_HOST` in the env.yml. Is it used anywhere else? If it is, I would have failed to change it, and if it's not, then I suppose it could just be cut from env.yml